docker/server.xml

```diff
@@ -43,8 +43,8 @@
         factory="org.apache.tomcat.jdbc.pool.DataSourceFactory"
         driverClassName="org.drizzle.jdbc.DrizzleDriver"
         url="${fineract_tenants_url}"
-        username="root"
-        password="mysql"
+        username="${username}"
+        password="${password}"
```

Comment:
Please use ’fineract_tenants_uid’ (and _pwd) instead of "username" and "password", to make more clear what these system properties are for.